src/include/sof/math/trig.h

```diff
@@ -13,6 +13,14 @@
 
 #include <stdint.h>
 
+#if defined __XCC__
+#include <xtensa/config/core-isa.h>
+#if XCHAL_HAVE_HIFI3 == 1
+#define CORDIC_HIFI3
+#include <xtensa/tie/xt_hifi3.h>
+#endif
+#endif
```

Comment:
it would be good to have these `endifs` commented simply because they are nested

Reply:
Yep, I just updated these two #endif lines.